content/en/docs/rules/fspath.md

```diff
@@ -0,0 +1,107 @@
+
+To help address this inconsistency, in Falco version XXX we added a new set of fields that normalize file system paths across a variety of syscalls.
+
+### `fs.path.*` fields
+
+The following fields are available for any syscall that operates on a file system path:
+
+* fs.path.name
+* fs.path.nameraw
+* fs.path.source
+* fs.path.sourceraw
+* fs.path.target
+* fs.path.targetraw
+
+`fs.path.name` is for file operations that work on a path like open, unlink, rmdir, etc. For other file operations that have a source and target, like cp, symlink, link, mv, etc, there are fields `fs.path.source` and `fs.path.target`.
+
+All of these convert relative paths to absolute paths when needed, using the thread's current working directory (cwd).
+
+`fs.path.nameraw/fs.path.sourceraw/fs.path.targetraw` are like the above but do *not* convert relative paths to absolute paths. They always contain the original path, which may or may not be relative.
+
+The fields only work for exit events and only return a value if the syscall was successful.
+
+The below tables show:
+* the specific syscalls that are are supported
+* the specific falco event numbers are supported
+* the specific event fields that are mapped to `fs.path.*` fields
+* whether the fields actually come from the enter event instead of the exit event.
+
+#### Single Argument File System Syscalls
+
+| Syscall  | Falco Libs Event Identifier   | `fs.path.name` field | From Enter Event?|
+| -------  | ------------------------------| ---------------------|------------------|
+| mkdir    | `PPME_SYSCALL_MKDIR`          | `evt.rawarg.path`    | Yes              |
+| mkdir    | `PPME_SYSCALL_MKDIR_2`        | `evt.rawarg.path`    | No               |
+| mkdirat  | `PPME_SYSCALL_MKDIRAT`        | `evt.rawarg.path`    | No               |
+| rmdir    | `PPME_SYSCALL_RMDIR`          | `evt.rawarg.path`    | Yes              |
+| rmdir    | `PPME_SYSCALL_RMDIR_2`        | `evt.rawarg.path`    | No               |
+| unlink   | `PPME_SYSCALL_UNLINK`         | `evt.rawarg.path`    | Yes              |
+| unlink   | `PPME_SYSCALL_UNLINK_2`       | `evt.rawarg.path`    | No               |
+| unlinkat | `PPME_SYSCALL_UNLINKAT`       | `evt.rawarg.name`    | Yes              |
+| unlinkat | `PPME_SYSCALL_UNLINKAT_2`     | `evt.rawarg.name`    | No               |
+| open     | `PPME_SYSCALL_OPEN`           | `evt.rawarg.name`    | No               |
+| openat   | `PPME_SYSCALL_OPENAT`         | `evt.rawarg.name`    | Yes              |
+| openat   | `PPME_SYSCALL_OPENAT_2`       | `evt.rawarg.name`    | No               |
```

Comment:
Explain to the reader why we sometimes have 2? When I write a rule with openat syscall which one is used? Same applies to the other instances.

Reply:
It was kind of covered in the introduction, but I moved it to the table introduction just as background.